### PR TITLE
Fixing logic to not show actions with no "passed" field in the final summary

### PR DESCRIPTION
**Pull request by gdelgadot**
Currently the GitHub Action is showing successful actions as failed because those actions have no `passed` field. From now on, those actions will not be shown in the final summary.

Signed-off-by: Gloria Delgado <gloriad@vmware.com>

### CODE REVIEW

**Comment by gdelgadot** [comment]
[Here](https://cp.production.k.bromelia.bitnami.net/v1/execution-graphs/262d3a75-1cd8-4ded-b0a4-22c39a3c23df/report) we can find an example with the error described previously. It shows the `helm-publish` action as failed action:
Screenshot 2022-08-10 at 18 19 07
From now on, with this fix, the GitHub Action will not show the actions with no `passed` field:
Screenshot 2022-08-10 at 18 18 44

File: src/main.ts [hunk 1]
```diff
@@ -318,7 +318,7 @@ export function prettifyExecutionGraphResult(executionGraphResult: Object): void
     if (task["tests"]) {
       core.info(
         `${ansi.bold(task["action_id"])} ${ansi.bold("action:")} ${
-          task["passed"] === true ? ansi.green("passed") : ansi.red("failed")
+          task["passed"] === true ? ansi.bold(ansi.green("passed")) : ansi.red("failed")
```
Comment (recena) [inline src/main.ts:321]: Unrelated change.

File: src/main.ts [hunk 2]
```diff
@@ -328,19 +328,17 @@ export function prettifyExecutionGraphResult(executionGraphResult: Object): void
     } else if (task["vulnerabilities"]) {
       core.info(
         `${ansi.bold(task["action_id"])} ${ansi.bold("action:")} ${
-          task["passed"] === true ? ansi.green("passed") : ansi.red("failed")
+          task["passed"] === true ? ansi.bold(ansi.green("passed")) : ansi.red("failed")
```
Comment (recena) [inline src/main.ts:331]: Ditto.

File: src/main.ts [hunk 3]
```diff
@@ -318,7 +318,7 @@ export function prettifyExecutionGraphResult(executionGraphResult: Object): void
     if (task["tests"]) {
```
Comment (recena) [inline src/main.ts:318]: IMO we should apply the filter here.

Comment (gdelgadot) [reply src/main.ts]: I made a change here, what do you think?

**Review by recena — APPROVED** [review]
